c/bpxskt.c

```diff
@@ -107,6 +107,28 @@
 
 #endif
 
+/* xlclang and clang what prototypes, these are incomplete, but quiet the compiler */
+int BPXSOC();
+int BPXCON();
+int BPXGHN();
+int BPXSLP();
+int BPXCHR();
```

Comment:
`BPXCHR` doesn't seem to be referenced anywhere.

Reply:
You must be a very careful reader!!